### DISCUSSION TITLE
Move each port install to a dedicated list/tab in Package Manager

### DISCUSSION THREAD

**Discussion by oherau** [OnionUI/Onion#1542] [Ideas]
Hi there,

For now ports must be installed manually, only the shortcut install and game import are automated.
It would be a great feature to display ports in a catalog such as in Linux distibutions (ex: synaptic) and allow to install them automatically from the official repo.

The best option I see for that in a new tab in the Package Manager called "PORTS", with the same icons as emulators: install YES/NO, resources/folder already present YES/NO
The port list should grow up so adding a dedicated tab would be a very nice feature.
In addition to that some ports might be tricky to install so automating it would be nice too.

**Comment by XK9274** [comment]
Hey,

An app or equivalent is planned 👍

Thanks

**Comment by oherau** [comment]
Cool ! I can't wait to test it!
